packages/w/wren/xmake.lua

```diff
@@ -0,0 +1,79 @@
+package("wren")
+    set_homepage("http://wren.io")
+    set_description("The Wren Programming Language. Wren is a small, fast, class-based concurrent scripting language.")
```

Comment:
Shorten description by removing the first sentence so it's just:
`Wren is a small, fast, class-based concurrent scripting language.`

Reply:
fixed